db-engine/src/main/java/com/splicemachine/db/impl/sql/catalog/DataDictionaryImpl.java

```diff
@@ -669,6 +669,33 @@ private void getBuiltinSystemSchemas(){
 
     }
 
+    public List<SchemaDescriptor> getAllSchemas(TransactionController tc) throws StandardException{
+        List<SchemaDescriptor> lists =new ArrayList<>();
```

Comment:
Could you change `lists` to `allSchemas` or `schemas` to make the name a bit more self-explanatory?